### PR TITLE
feat: add HTML report without JS

### PR DESCRIPTION
**Pull request by jeremylong**
## Fixes Issue #5039

Add a JENKINS report type - which is a copy of the HTML report without the JavaScript. As such the HTML Report for Jenkins would not have the capability to create suppression rules.

### CODE REVIEW

File: ant/src/main/java/org/owasp/dependencycheck/taskdefs/Check.java [hunk 1]
```diff
@@ -229,12 +229,12 @@ public class Check extends Update {
...
     private Boolean autoUpdate;
     /**
-     * The report format to be generated (HTML, XML, JUNIT, CSV, JSON, SARIF,
+     * The report format to be generated (HTML, XML, JUNIT, CSV, JSON, SARIF, JENKINS,
      * ALL). Default is HTML.
      */
     private String reportFormat = "HTML";
     /**
-     * The report format to be generated (HTML, XML, JUNIT, CSV, JSON, SARIF,
+     * The report format to be generated (HTML, XML, JUNIT, CSV, JSON, SARIF, JENKINs,
```
Comment (nhumblot) [inline ant/src/main/java/org/owasp/dependencycheck/taskdefs/Check.java]: Hi 👋

Shouldn't it be a caps locked `JENKINS` value instead? 🤔

**Review by nhumblot — APPROVED** [review]
Hi 👋

Looks good to me except a typo. I also went to look into the website to see if there would be some place where the documentation should be changed but I don't think so.

Thank you for your work! 🤗

File: ant/src/main/java/org/owasp/dependencycheck/taskdefs/Check.java (hunk 1 above)
Comment (jeremylong) [reply ant/src/main/java/org/owasp/dependencycheck/taskdefs/Check.java]:
```suggestion
     * The report format to be generated (HTML, XML, JUNIT, CSV, JSON, SARIF, JENKINS,
```

**Comment by jeremylong** [comment]
Thanks for catching the typo!

**Pull request merged**